styles.css

```diff
@@ -122,7 +122,8 @@ button:hover {
 }
 
 .new-idea {
-  display: inline-block;
+  display: flex;
```

Comment:
@ben-rosner-williamsburg Is there a particular reason you changed this to `display: flex`? I struggled back and forth with this for while and landed on `display: inline-block`, so if you have insight as to why flex works better, I would love the help in understanding it!

Reply:
@ericahagle It made positioning the delete button much easier. I'm hesitant to switch back to inline-block for this reason.

Reply:
Fair! I changed it in the PR I'm working on, too, and it's been peachy for positioning the fave buttons, too. Good call! 🌮